src/pages/documentation/platform-assurance/get-access.md

```diff
@@ -2,9 +2,9 @@
 
 <InlineAlert variant="info" slots="text"/>
 
-## Project Griffon will be Assurance!
+## Project Griffon is now Assurance
 
-We're pleased to announce that Project Griffon will be generally available to all Adobe Experience Cloud customers as Assurance. To learn more about this transition see [here](./index.md).
+We're pleased to announce that Project Griffon is now generally available to all Adobe Experience Cloud customers as Assurance. To learn more about this transition see [here](./index.md).
```

Comment:
Do we need to keep the Assurance banner on get access? If so, then change it to the phrasing I had above.